apps/api/src/server.ts

```diff
@@ -52,6 +53,7 @@ const startServer = async () => {
 	app.use('/signature', signatureRouter);
 	app.use('/auth', authRouter);
 	app.use('/file', fileRouter);
+	app.use(errorHandler);
```

Comment:
Error handler needs to be the final middleware/request handler added to the app so that all unhandled thrown from other routers/middleware are caught and handled.